<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGraphicsView" name="graphic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>650</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : none ;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::DontWrapRows</enum>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font-size:15px ;
font-weight: bold ;
text-align  : center ;</string>
       </property>
       <property name="text">
        <string>Parcours</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Partir de l'arrêt...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="starttop"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Aller à l'arrêt...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QComboBox" name="endtop"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="startApplication">
       <property name="text">
        <string>Démarrer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>180</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font-size:15px ;
font-weight: bold ;
text-align  : center ;</string>
       </property>
       <property name="text">
        <string>Arrêts</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Arrêt actuellement survolé</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="hoveredTop">
       <property name="text">
        <string>Aucun</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="hoveredTopId">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>270</y>
      <width>181</width>
      <height>359</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font-size:15px ;
font-weight: bold ;
text-align  : center ;</string>
       </property>
       <property name="text">
        <string>Voies</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>La voie sélectionnée relie les arrêts</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="activeArcTop1">
       <property name="text">
        <string>Aucun</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="activeArcTop2">
       <property name="text">
        <string>Aucun</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="activeArcId">
       <property name="styleSheet">
        <string notr="true">font-size:13px ;
font-weight: bold ;
text-align  : center ;</string>
       </property>
       <property name="text">
        <string>A &gt; B</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Existant : </string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Poids :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Phéromones :</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>En service</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="activeArcButton">
       <property name="text">
        <string>Désactiver</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="activeInverseArcId">
       <property name="styleSheet">
        <string notr="true">font-size:13px ;
font-weight: bold ;
text-align  : center ;</string>
       </property>
       <property name="text">
        <string>B &gt; A</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Existant : </string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Poids :</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Phéromones :</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>En service</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="2">
      <widget class="QPushButton" name="activeInverseArcButton">
       <property name="text">
        <string>Désactiver</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="activeArcExists">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="activeArcWeight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="activeArcPheromones">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="activeArcAvailable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="activeInverseArcExists">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="activeInverseArcWeight">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="activeInverseArcPheromones">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="activeInverseArcAvailable">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>249</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
